server/webhook.go

```diff
@@ -970,59 +996,118 @@ func (s *Server) prepareIssueFromSDLFile(ctx context.Context, repo *api.Reposito
 	return migrationDetailList, nil
 }
 
+type migrationFileYAMLDatabase struct {
```

Comment:
Consider moving this to api package since 1) this is API, 2) we can give some examples about it, and 3) test pckage can use this API.

Reply:
Addressed in https://github.com/bytebase/bytebase/pull/3307/commits/f1fa8e7926c089699de27c94d314d9d7d194e3de